index.html

```diff
@@ -17,81 +18,124 @@ <h2 class="article-title">{{ title }}</h2>
         <p class="article-caption">{{ caption }}</p>
       </div>
       <div class="article-body">{{{ body }}}</div>
-      <button type="button" class="read-more">Read More</button>
+      <p class="read-more">read more</p>
     </article>
   </script>
 </head>
 <body class="fade-out">
   <header>
-    <nav>
-      <ul>
-        <li id="menuIcon">&#9776;</li>
-        <li class="menuOption" data-content="blogContainer"><a href="#">Home</a></li>
-        <li class="menuOption" data-content="about"><a href="#">About</a></li>
-        <li class="menuLink"><a href="https://github.com/christylaguardia/" target="_blank">GitHub</a></li>
-        <li class="menuOption" data-content="contact"><a href="#">Contact</a></li>
-      </ul>
-    </nav>
+    <nav id="top-nav" class="menu">
+      <!-- TODO: get the logo to work -->
+      <!-- <div id="logo">
+      <div>
+      <img id="headshot-small" src="images/christy-headshot.jpeg">
+    </div>
+    <div>
+    <h3>Christy La Guardia</h3>
+    <p>Full Stack JavaScript Developer</p>
+  </div>
+</div> -->
+
+<ul class="menu">
+  <li id="menu-icon">&#9776;</li>
+  <li class="menu-option" data-content="home"><a href="#">Home</a></li>
+  <li class="menu-option" data-content="about"><a href="#">About</a></li>
+  <li class="menu-option" data-content="blog"><a href="#">Blog</a></li>
+  <li class="menu-option" data-content="projects"><a href="#">Projects</a></li>
+  <li class="menu-option" data-content="hire-me"><a href="#">Hire Me</a></li>
+</ul>
+</nav>
+</header>
+
+<section id="home" class="tab-content">
+  <article>
     <img id="headshot" src="images/christy-headshot.jpeg">
     <h1>Christy La Guardia</h1>
-    <p>Beagle lover, wannabe coder and ex-JW.</p>
-  </header>
+    <!-- <p>Beagle lover, wannabe coder and ex-JW.</p> -->
+    <h3>Full Stack JavaScript Developer</h3>
+    <ul>
+      <li class="menu-option" data-content="about"><a href="#">About</a></li>
+      <li class="menu-option" data-content="blog"><a href="#">Blog</a></li>
+      <li class="menu-option" data-content="projects"><a href="#">Projects</a></li>
+      <li class="menu-option" data-content="hire-me"><a href="#">Hire Me</a></li>
+      <li>
+        <i class="fa fa-envelope" aria-hidden="true"></i>
+        <a href="mailto:christinelaguardia@gmail.com"> Email</a>
+      </li>
+      <li>
+        <i class="fa fa-phone" aria-hidden="true"></i>
+        <a href="tel:5034071529"> Call/Text</a>
+      </li>
+      <li>
+        <i class="fa fa-github" aria-hidden="true"></i>
+        <a href="https://github.com/christylaguardia/" target="_blank">GitHub</a>
+      </li>
+      <li>
+        <i class="fa fa-linkedin-square" aria-hidden="true"></i>
+        <a href="www.linkedin.com/in/christy-la-guardia"> LinkedIn</a>
```

Comment:
This link gave me problems... please check it? Also, it is not opening in a new, separate tab, which I prefer for these type links... :)